services/web/server/src/simcore_service_webserver/exporter/archiving.py

```diff
@@ -115,12 +98,15 @@ async def zip_folder(project_id: str, input_path: Path, no_compression=False) ->
 
 
 async def unzip_folder(input_path: Path) -> Path:
-    command_args = ["unzip", str(input_path), "-d", str(input_path.parent)]
-
-    await _wrap_create_subprocess_exec(
-        command_args=command_args,
-        cwd_path=input_path.parent,
-        fail_message=f"Could not decompress {str(input_path)}",
-    )
+    try:
+        await run_in_process_pool(
+            shutil.unpack_archive,  # callable
+            str(input_path),  # filename
+            str(input_path.parent),  # extract_dir
+            "zip",  # format
+        )
+    except Exception as e:
+        log.exception(e)
+        raise ExporterException(str(e)) from e
```

Comment:
same here

Reply:
Same as above. Will just refactor to only have this in a single place.